jenkins/release-workflows/release-chores.jenkinsfile

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+lib = library(identifier: 'jenkins@8.2.0', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/opensearch-project/opensearch-build-libraries.git',
+]))
+
+pipeline {
+    options {
+            timeout(time: 1, unit: 'HOURS')
+    }
+    agent {
+        docker {
+            label 'Jenkins-Agent-AL2023-X64-M54xlarge-Docker-Host'
+            image 'opensearchstaging/ci-runner:ci-runner-centos7-opensearch-build-v3'
+            registryUrl 'https://public.ecr.aws/'
+            alwaysPull true
+        }
+    }
+    parameters {
+        string(
+            name: 'RELEASE_VERSION',
+            description: "Required: On-going release version, e.g. 2.0.0",
+            trim: true
+        )
+        string(
+            name: 'RELEASE_CHORE',
+            description: "Required: What release chore you wanna do? e.g: 'add_rc_details_comment'",
+            trim: true
+        )
+    }
+    stages {
+        stage('Parameter check') {
+            steps {
+                script {
+                    if (!(RELEASE_VERSION && RELEASE_CHORE)) {
+                        error('Required parameters are missing. Please provide the mandatory arguments RELEASE_VERSION and RELEASE_CHORE')
+                    }
+                }
+            }
+        }
+        stage('Add RC details and Docker Scan comment') {
+            when {
+                expression { params.RELEASE_CHORE == 'add_rc_details_comment' }
+            }
+            steps {
+                script {
+                    addRcDetailsComment(
```

Comment:
Wonder will this library help collapse/cross the previous RC comments?

Reply:
Nope it won't. We dont do that even today. Check the sample comment in the PR description. This will just add new RC comment. Wont touch anything else.

Reply:
I think this would be a nice-to-have feature if feasible. We used to do it manually at least update the previous RC comments to indicate they are no longer valid. We've seen issues when teams were not using the latest RC.
Not a blocker for this PR but maybe we can keep track of this as enhancement.